modules/currency_denomination/includes/commerce_pos_currency_denomination.admin.inc

```diff
@@ -0,0 +1,361 @@
+    '#value' => t('Save settings'),
+  );
+
+  $form['#attached']['css'][] = drupal_get_path('module', 'commerce_pos_currency_denomination') . '/css/commerce_pos_currency_denomination.css';
+
+  return $form;
+}
+
+/**
+ * Submit handler for the POS currency denomination settings form.
+ */
+function commerce_pos_currency_denomination_settings_form_submit($form, &$form_state) {
+  if (!empty($form_state['values']['commerce_pos_default_currency_denomination_types'])) {
+    variable_set('commerce_pos_default_currency_denomination_types', $form_state['values']['commerce_pos_default_currency_denomination_types']);
+  }
+}
+
+/**
+ * The form for adding new denomination types.
+ */
+function commerce_pos_currency_add_denomination_type_form($form, &$form_state) {
+  $form['commerce_pos_denomination_type_currency_code'] = array(
+    '#type' => 'textfield',
+    '#size' => 3,
+    '#maxlength' => 3,
+    '#default_value' => NULL,
+    '#title' => t('Denomination Type Currency Code'),
+    '#description' => t('Currency code should be in the following format: "USD".'),
+    '#required' => TRUE,
+  );
+
+  $form['actions']['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save'),
+  );
+
+  return $form;
+}
+
+/**
+ * Validate handler for adding new denomination types.
+ */
+function commerce_pos_currency_add_denomination_type_form_validate($form, &$form_state) {
+  // Check if the denomination type already exists in the db.
+  $result = db_select('commerce_pos_currency_denomination_type', 't')
+    ->fields('t', array('currency_id'))
+    ->condition('currency_code', $form_state['values']['commerce_pos_denomination_type_currency_code'])
+    ->execute()
+    ->fetchField();
+
+  if ($result) {
+    form_set_error('', t('This denomination type already exists in the system.'));
+  }
+}
+
+/**
+ * Submit handler for adding new denomination types.
+ */
+function commerce_pos_currency_add_denomination_type_form_submit($form, &$form_state) {
+  db_insert('commerce_pos_currency_denomination_type')
+    ->fields(array(
+      'currency_code' => $form_state['values']['commerce_pos_denomination_type_currency_code']
+    ))
+    ->execute();
+
+  drupal_set_message(t('The denomination type has been successfully added.'));
+  drupal_goto('admin/commerce/config/pos/denomination');
+}
+
+/**
+ * Delete a denomination type.
+ */
+function commerce_pos_currency_delete_denomination_type_form($form, &$form_state, $currency_id) {
+  // Get the denomination type info.
+  $form_state['denomination_type_info'] = commerce_pos_denomination_get_currency_types_from_id(array($currency_id));
+
+  if (!empty($form_state['denomination_type_info'])) {
+    $form_state['denomination_type_info'] = $form_state['denomination_type_info'][$currency_id];
+
+    return confirm_form(
+      $form,
+      t('Are you sure you want to delete currency denomination type @type?', array(
+        '@type' => $form_state['denomination_type_info']->currency_code

```

Comment:
All lines in a multi-line array should have a comma, even the last element: https://www.drupal.org/docs/develop/standards/coding-standards#array